data_diff/diff_tables.py

```diff
@@ -120,25 +137,45 @@ def _get_stats(self) -> DiffStats:
         table2_count = self.info_tree.info.rowcounts[2]
         unchanged = table1_count - diff_by_sign["-"] - diff_by_sign["!"]
         diff_percent = 1 - unchanged / max(table1_count, table2_count)
+        if is_dbt:
+            return DiffStats(diff_by_sign, table1_count, table2_count, unchanged, diff_percent, extra_column_diffs)
+        else:
+            return DiffStats(diff_by_sign, table1_count, table2_count, unchanged, diff_percent, None)
```

Comment:
if you add `extra_column_diffs = None` at the top, this can be simplified
```suggestion
        return DiffStats(diff_by_sign, table1_count, table2_count, unchanged, diff_percent, extra_column_diffs)
```